benchmark/sirun/appsec-iast/meta.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "appsec-iast",
+  "cachegrind": false,
+  "instructions": true,
+  "iterations": 40,
```

Comment:
Reduced from 100 to 40 based on this: https://github.com/DataDog/dd-trace-js/blob/839a6f8e75e2642ed107aa90c168681c1052a07c/benchmark/sirun/plugin-http/meta.json#L4